crates/ln-dlc-node/src/node/dlc_channel.rs

```diff
@@ -371,6 +449,36 @@ impl<S: TenTenOneStorage + 'static, N: LnDlcStorage + Sync + Send + 'static> Nod
                 .sum(),
         ))
     }
+
+    /// Check if it is safe to close the DLC channel collaboratively.
+    ///
+    /// In general, it is NOT safe to close an DLC channel if there still is a position attached
+    /// to it. This is because this can lead to loss of funds.
```

Comment:
🤔 I thought this only applied to Lightning channels.

With a split Lightning channel, if we triggered a collaborative close via LDK it would fail and the channel would be force-closed (IIRC). But here we are only operating with `rust-dlc`.

I'm surprised collab-close doesn't work in more states (particularly `Established`), since the API asks for the `accept_settlement_amount` anyway.

Reply:
I honestly haven't tried... 🙈

Reply:
See here: https://github.com/get10101/10101/pull/1806#discussion_r1445795987: if there is still an open DLC we would need to close the counter position as well or hedge otherwise. I've updated the doc and will leave it as is.